script/avalon_push_images

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+SOURCE=$1
+DESTINATION=$2
+TAGS_CSV=$3
+TAGS=()
+COMMAND_BASE="podman image push "
+
+IFS=, TAGS=($TAGS_CSV)
+IFS=" "
+
+echo "TAGS"
+echo ${TAGS[@]}
+
+if [ "${TAGS}" == "" ]; then
+  echo "No tags found; exiting"
+  exit 1
+fi
+
+for tag in "${TAGS[@]}"; do
+  echo "tag '$tag'"
+  #exit 1 if $tag.empty?
+  command="${COMMAND_BASE} ${SOURCE} ${DESTINATION}:${tag}"
+  echo "pushing image ... command:"
+  echo $command
+  #`$command`
```

Comment:
Do you plan on enabling this or will this script only return the commands as guidance to be copy/paste'd?

Reply:
Yes, I do want to enable it. maybe we need a --dry-run option or something to see what it's going to do without actually executing the command.

Reply:
:+1: I like the idea of a `--dry-run`.